### ISSUE TITLE
window is not defined in Nuxt JS

### ISSUE THREAD

**Issue by ChunAllen** [dbrekalo/vue-date-pick#5]
I install this plugin using **npm install vue-date-pick**

then I created a datepicker.js under plugins folder
```
import Vue from 'vue';
import DatePick from 'vue-date-pick';

Vue.use(DatePick);
```

and import it in **nuxt-config.js**
```
 plugins: [
    '~/plugins/datepicker'
  ],
```

then when I use it inside the template
**index.vue**
```
 <date-pick v-model="date"></date-pick>

<script>
import DatePick from 'vue-date-pick';
import 'vue-date-pick/dist/vueDatePick.css';

export default {
    components: {DatePick},
    data: () => ({
        date: '2019-01-01'
    })
};
</script>
```

I'm getting an error **window is not defined**

**Comment by dbrekalo** [comment]
@ChunAllen i think there is no need to register it as plugin - would you try if it works without it?

**Comment by dbrekalo** [comment]
this is related to https://github.com/webpack/webpack/issues/6522

**Comment by dbrekalo** [comment]
@ChunAllen fixed in 1.0.7
